package.json

```diff
@@ -8,6 +8,7 @@
     "lint:md": "markdownlint **/*.md --ignore node_modules",
     "lint:ts": "eslint --color --ext .ts,.js .",
     "lint:yaml": "yamllint --ignore=node_modules/**/*.yaml **/*.yaml",
+    "test": "npm run compile:gameStateSchema && npm run test:src && npm run test:e2e",
```

Comment:
For local development

Reply:
Should this also run `typecheck`?

Reply:
This is for running the tests. We can make a command that runs typecheck, lint and `test` when/if we need to